main.go

```diff
@@ -81,40 +78,18 @@ func main() {
 	flag.StringVar(&defaultEmail, "default-email", "", "Default email address for ACME registrations")
 	flag.IntVar(&renewBeforeDays, "renew-before-days", 7, "Renew certificates before this number of days until expiry")
 	flag.IntVar(&workers, "workers", 4, "Number of parallel jobs to run at once")
+	flag.StringVar(&dbArgs, "db-args", "host=localhost port=5432 user=certmanager dbname=certmanager password=Pass1234 sslmode=disable",
+		"db host, port, username, password, sslmode")
```

Comment:
This feels a little awkward. Would it be be better to have `db-host`, `db-user`, etc..?  Down the road we should think about moving to a configuration system like [viper](https://github.com/spf13/viper), but that's likely outside the scope of this effort.